docs/reference/utility-apis/IdentityApi.md

```diff
@@ -1,7 +1,7 @@
 # IdentityApi
 
 The IdentityApi type is defined at
-[packages/core-api/src/apis/definitions/IdentityApi.ts:22](https://github.com/backstage/backstage/blob/ca535f2f66c3a4980c80f4b1a049dfd07569010e/packages/core-api/src/apis/definitions/IdentityApi.ts#L22).
+[packages/core-api/src/apis/definitions/IdentityApi.ts:22](https://github.com/backstage/backstage/blob/abd85269fbea466c18a71fa661335c0713a32c07/packages/core-api/src/apis/definitions/IdentityApi.ts#L22).
```

Comment:
These are generated, use `yarn docgen` instead

Reply:
oh TIL, there were a lot more then! @Rugvip I wonder if it that command should automatically get ran in some way?

Reply:
But surprised the links broke tbh, they should be stable :p

Most of the updates are just switching the links to the master sha, which isn't really needed. It just needs updates when changing the core apis